php/tests/WellKnownTest.php

```diff
@@ -367,12 +367,28 @@ public function testWrappers()
 
         $m = new Int64Value();
         $m->setValue(1);
-        $this->assertEquals(1, $m->getValue());
+        $this->assertSame(1, $m->getValue());
```

Comment:
This is breaking on 32-bit, it's returning '1'

Reply:
@mkruskal-google hmm, yes.
As I wrote before, 
I think any which can fit to int32 also should be converted to int. I'll fix.

Reply:
UPD: Reverted because that `GPBUtil::checkInt64` isn't in PR scope.

But some `GPBUtil` methods seems outdated for PHP 8.0 (currently min supported version), I'll try to improve it a bit later. :-)

Reply:
yea this code really confuses me, and I'm not saying we shouldn't clean it up.  Just that it's better to keep PRs targetted

Reply:
looks like there's still 3 issues on 32-bit

Reply:
Okay, I'm downloading 32-bit debian image 🌚.

Reply:
My code is almost no longer able to run properly on a 64-bit machine. 2147483648 => -2147483648, my timestamp cannot exceed 2038-01-19 11:14:08, I hope it can be fixed soon. Thanks

@mkruskal-google